.dockerignore

```diff
@@ -1,3 +1,10 @@
-data
 interface
 adage/adage/config.py
+
+**/*~
+**/*.pyc
```

Comment:
So it is a wildcard string "that matches any number of directories (including zero)" (https://docs.docker.com/engine/reference/builder/#dockerignore-file). It basically ignores any files ending in `~` and `.pyc` in the current folder or any of the subdirectories.